bindings/python/Python.cc

```diff
@@ -94,6 +94,7 @@ class TokenizerWrapper
                    bool segment_case,
                    bool segment_numbers,
                    bool segment_alphabet_change,
+                   bool support_prior_joiners,
```

Comment:
This argument should also be added to the Python interface declaration (line 363 and after).

Reply:
done